hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -147,7 +147,7 @@ protected void runTask() throws Exception {
           unhealthyContainerStateStatsMap);
       long start = Time.monotonicNow();
       long currentTime = System.currentTimeMillis();
```

Comment:
currentTime is still using System.currentTimeMillis().  It will be passed to `ContainerHealthRecords.generateUnhealthyRecords(`.  Not sure if it should be changed to `Time.monotonicNow()`.